gpii/node_modules/processReporter/src/processesBridge.js

```diff
@@ -36,61 +36,52 @@ fluid.defaults("gpii.processes", {
     invokers: {
         findSolutionsByCommands: {
             funcName: "gpii.processes.findSolutionsByCommands",
-            args: ["{that}", "{arguments}.0"]
-                             // array of command names
+            args: ["{that}", "{arguments}.0"] // array of command names
         },
         findSolutionsByPids: {
             funcName: "gpii.processes.findSolutionsByPids",
-            args: ["{that}", "{arguments}.0"]
-                             // array of pids (process ids)
```

Comment:
These indenting changes are unacceptable since the purpose of the commenting is to line up with the position of the start of the first interpolated argument. Is there a way we can disable indenting checks for comments?

Reply:
Yes, just testing it locally with universal before moving it into the defaults.

Reply:
Turns out it's way too naive, it expects C-style comments only and I couldn't get it to work with the option they provide to pass a regular expression.  As discussed with Antranig, I ended up limiting the indentation check to JSON files, which we have no other way to lint.